narayana-release-process.sh

```diff
@@ -126,9 +110,13 @@ then
     exit
   fi
   set -e
-  echo "Checking if there were any failed jobs, this may be interactive so please stand by"
-  JIRA_HOST=issues.redhat.com JENKINS_JOBS=narayana,narayana-catelyn,narayana-documentation,narayana-hqstore,narayana-jdbcobjectstore,narayana-quickstarts,narayana-quickstarts-catelyn\
-      ./scripts/release/pre_release.py
+  
+  read -p "Please check for Jira Blocker issues with Fix.Version set to {version}.next which are not yet resolved(https://issues.redhat.com/issues/?jql=project%20%3D%20JBTM%20AND%20priority%20%3D%20Blocker%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC) and failed CI jobs before continuing. Continue? y/n " NOBLOCKERS
```

Comment:
I don't think we set {version}.next until the fix is ready so that part isn't quite what I would expect someone be able to do today

Reply:
Thanks Tom, I'll update that line. Thanks for raising JBTM-3891